.github/workflows/hip.yml

```diff
@@ -46,20 +46,12 @@ jobs:
         python3 -m pip install -U build packaging setuptools[core] wheel
         python3 -m pip install -U cmake
 
-        # "mpic++ --showme" forgets open-pal in Ubuntu 20.04 + OpenMPI 4.0.3
-        #   https://bugs.launchpad.net/ubuntu/+source/openmpi/+bug/1941786
-        #   https://github.com/open-mpi/ompi/issues/9317
-        export LDFLAGS="-lopen-pal"
-
         cmake -S . -B build              \
             -DCMAKE_VERBOSE_MAKEFILE=ON  \
+            -DpyAMReX_IPO=OFF            \
             -DAMReX_GPU_BACKEND=HIP      \
-            -DAMReX_SPACEDIM="1;2;3"     \
-            -DCMAKE_C_COMPILER=$(which clang)             \
-            -DCMAKE_CXX_COMPILER=$(which clang++)         \
-            -DAMReX_AMD_ARCH=gfx1010                      \
-            -DAMReX_ROCTX=ON                              \
-            -DCMAKE_CXX_STANDARD=17
+            -DAMReX_AMD_ARCH=gfx90a      \
```

Comment:
In WarpX, https://github.com/BLAST-WarpX/warpx/blob/development/.github/workflows/hip.yml, we seem to have `-DAMReX_AMD_ARCH=gfx900` instead of `-DAMReX_AMD_ARCH=gfx90a`. Do you happen to know why and/or if that matters at all?

Reply:
I don't think it matters at all. gfx90a is what's on frontier. gfx900 is quite old.

Reply:
I know for sure is that gfx90a with IPO does not work.